PixivUtil2.py

```diff
@@ -1023,9 +1023,9 @@ def menu_fanbox_download_from_artist(op_is_valid, via, args):
         PixivHelper.print_and_log("info", f"No {via_type} artist!")
         return
     PixivHelper.print_and_log("info", f"Found {len(artists)} {via_type} artist(s)")
-    print(", ".join(str(artists)))
 
     for artist in artists:
+        __config__.loadConfig(path=configfile)
         __br__.fanboxUpdateArtistToken(artist)
```

Comment:
any reason not to move it inside processFanboxArtist()? it is called in menu_fanbox_download_from_artist() and menu_fanbox_download_by_artist_or_creator_id, so at least the behaviour is consistent for those menu.

Reply:
well, briefly speaking, it's just simpler.
specifically speaking:
1. the first parameter for `processFanboxArtist` is a `FanboxAritst` class item
2. another menu to download fanbox with id also calls `processFanboxArtist` after it gets the `FanboxArtist` instance with browser `fanboxGetArtistById` inside which calls the `fanboxUpdateArtistToken` method
3. currently when downloading from supporting list or following list, the menu method first gets a list of `FanboxArtist` instance with `fanboxGetUsers` method, and then passes each artist instance to `processFanboxArtist`, in which it calls the `fanboxUpdateArtistToken` to update some other fields of the instance before proceeding with the posts.

So if i need to put config.loadConfig inside `fanboxUpdateArtistToken`, I need to:
1. make `Fanbox.parseArtists` return a list of ids
2. change the first parameter of `processFanboxArtist` to an id instead of `FanboxAritst` instance
3. put `config.loadConfig` inside `fanboxUpdateArtistToken`
4. put `artist = __br__.fanboxGetArtistById` inside `fanboxUpdateArtistToken` after 3.
5. delete calling of `__br__.fanboxGetArtistById` and `__br__.fanboxUpdateArtistToken` in related menu methods
6. preferably change the name for some methods to make the name clear and match its content

or there could be some other ways as well.... but it's not that it's difficult, just too much fuss....

and I from the top am against that the config should reload between artists. it's like not a regular operation.
it's like being given a list of jobs, and instead of having the same criteria, or a set of criteria for each at the start, the giver calls in and checks on you and changes the criteria in between the jobs. or that a program is running and someone wants to cut in and change the value of one parameter.
this kind of case exists and makes sense but it's not how it should be.
i think the users would always have all different kinds of ideas, sometimes absurd. maybe next time he would say, the process of one creator is too fast, please insert a prompt in between the creators so i have time to change the configuration. sometimes mine are absurd too. 

anyway, i am just venting. if you would like to have them changed to the later form, please let me know.